src/dmd/mtype.d

```diff
@@ -4922,6 +4922,23 @@ extern (C++) final class TypeFunction : TypeNext
         return MATCH.nomatch;
     }
 
+    /**
+     * Check if the arguments of a call are possibly inverted and prints an
+     * additional error message if the call would compile.
+     */
+    extern (D) void checkInvertedMatch(Type tthis, Expressions* args, int flag = 0, const(char)** pMessage = null, Scope* sc = null)
```

Comment:
- All calls use `null` for `tthis`
- Document parameters in DDOC
- No default parameter
- `flags` is always `0`
- `pMessage` is not used

Reply:
thanks, fixed.